network/proxy.md

```diff
@@ -26,7 +26,7 @@ configure it in different ways:
 
     You can optionally exclude hosts or ranges from going through the proxy
     server by setting a `noProxy` key to one or more comma-separated IP
-    addresses or hosts. Using the `*` character as a wildcard for hosts and using CIDR notation for IP addresses is supported as
+    addresses or hosts. Prefixing domain with a dot `.` acts as a wildcard for hosts and using CIDR notation for IP addresses is supported as
```

Comment:
Hm.. yes, these are "complicated", as the supported formats are not formalised. Perhaps we should mention something about that (supported formats depend on the software that's using the `NO_PROXY` (and other `_PROXY`/ `_proxy` vars), and we could link to the Go documentation, and possibly copy some of that; https://pkg.go.dev/golang.org/x/net/http/httpproxy#Config

```
NoProxy represents the NO_PROXY or no_proxy environment
variable. It specifies a string that contains comma-separated values
specifying hosts that should be excluded from proxying. Each value is
represented by an IP address prefix (1.2.3.4), an IP address prefix in
CIDR notation (1.2.3.4/8), a domain name, or a special DNS label (*).
An IP address prefix and domain name can also include a literal port
number (1.2.3.4:80).
A domain name matches that name and all subdomains. A domain name with
a leading "." matches subdomains only. For example "foo.com" matches
"foo.com" and "bar.foo.com"; ".y.com" matches "x.y.com" but not "y.com".
A single asterisk (*) indicates that no proxying should be done.
A best effort is made to parse the string and errors are
ignored.
```

I seem to recall we did that somewhere (but it could be I started working on it, but never pushed a PR 🤔)